### DISCUSSION TITLE
weblate github pull request on private repo

### DISCUSSION THREAD

**Discussion by zisuu** [WeblateOrg/weblate#9407] [Q&A] [answered]
Hello

I try to add a github enterprise private repo in weblate as a new translation component. I need to use the https git url since ssh outbound is blocked in our environment.

From my understanding there are two options how you can use github private repos:
1) with component type "Git" and https url inkl. credetianls, like here: https://docs.weblate.org/en/latest/vcs.html#https-repositories
2) with component type "Github Pull Request" and https url - but not with credentials in the url (I guess?) because we set the https://docs.weblate.org/en/latest/admin/config.html#std-setting-GITHUB_CREDENTIALS for this.

the following things are unclear to me:
- If i use Option 2, do I need to use the default https git url without credentials in it, or do I need to use the same git https url incl. credentials, like in Option 1?
Thanks a lot!

**Comment by nijel** [comment, marked as answer]
The option 2 still needs URL to clone and push the repo, so you need same as in option 1.

Actually, it is even worse, because the pull request with forking, will always use SSH URL to push to the fork, so if SSH is not working, it won't work.

**Reply by zisuu** [reply]
Thanks a lot Michal!
